docker-compose.yml

```diff
@@ -83,10 +82,6 @@ services:
       - traefik.http.routers.activemq_http.entrypoints=http
   alpaca:
     image:  ${REPOSITORY:-local}/alpaca:latest
-    volumes:
-      - karaf-data:/opt/karaf/data
-    depends_on:
-      - activemq
```

Comment:
Don't we want to keep this?

Reply:
It doesn't really have much of an affect aside from when running ``docker-compose up alpaca`` causes `activemq` to start as well, it doesn't guarantee startup order or anything. I don't think we need it.